.github/workflows/manual-e2e.yml

```diff
@@ -5,22 +5,99 @@ on:
   workflow_dispatch:
 
 jobs:
-  manual-integration-main:
+  happy-path-test:
     runs-on: ubuntu-latest
-    timeout-minutes: 60
+    timeout-minutes: 20
     steps:
       - uses: actions/setup-go@v4
         with:
           go-version: "1.20"
       - uses: actions/checkout@v3
-
       - name: Checkout LFS objects
         run: git lfs checkout
-
       - name: Setup Go
         uses: actions/setup-go@v4
         with:
           go-version: "1.20" # The Go version to download (if necessary) and use.
-
-      - name: E2E tests
-        run: make test-e2e
+      - name: E2E happy-path test
+        run: go run ./tests/e2e/... --tc happy-path
+  changeover-test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    steps:
+      - uses: actions/setup-go@v4
+        with:
+          go-version: "1.20"
+      - uses: actions/checkout@v3
+      - name: Checkout LFS objects
+        run: git lfs checkout
+      - name: Setup Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: "1.20" # The Go version to download (if necessary) and use.
+      - name: E2E changeover test
+        run: go run ./tests/e2e/... --tc changeover
```

Comment:
Looks good, can you also check `Makefile` commands?

If you have time, consider refactoring the Makefile commands listed below. You're free to change or remove the commands as you see fit:
```Makefile

# run only happy path E2E tests
test-e2e-short:
	go run ./tests/e2e/... --happy-path-only

# run only happy path E2E tests with cometmock
# this set of traces does not test equivocation but it does check downtime
test-e2e-short-cometmock:
	go run ./tests/e2e/... --cometmock-happy-path --use-cometmock --use-gorelayer

# run full E2E tests in sequence (including multiconsumer)
test-e2e-multi-consumer:
	go run ./tests/e2e/... --include-multi-consumer

# run full E2E tests in parallel (including multiconsumer)
test-e2e-parallel:
	go run ./tests/e2e/... --include-multi-consumer --parallel

# run full E2E tests in sequence (including multiconsumer) using latest tagged gaia
test-gaia-e2e:
	go run ./tests/e2e/... --include-multi-consumer --use-gaia

# run only happy path E2E tests using latest tagged gaia
test-gaia-e2e-short:
	go run ./tests/e2e/... --happy-path-only --use-gaia

# run full E2E tests in parallel (including multiconsumer) using latest tagged gaia
test-gaia-e2e-parallel:
	go run ./tests/e2e/... --include-multi-consumer --parallel --use-gaia

# run full E2E tests in sequence (including multiconsumer) using specific tagged version of gaia
# usage: GAIA_TAG=v9.0.0 make test-gaia-e2e-tagged
test-gaia-e2e-tagged:
	go run ./tests/e2e/... --include-multi-consumer --use-gaia --gaia-tag $(GAIA_TAG)

# run only happy path E2E tests using latest tagged gaia
# usage: GAIA_TAG=v9.0.0 make test-gaia-e2e-short-tagged
test-gaia-e2e-short-tagged:
	go run ./tests/e2e/... --happy-path-only --use-gaia --gaia-tag $(GAIA_TAG)

# run full E2E tests in parallel (including multiconsumer) using specific tagged version of gaia
# usage: GAIA_TAG=v9.0.0 make test-gaia-e2e-parallel-tagged
test-gaia-e2e-parallel-tagged:
	go run ./tests/e2e/... --include-multi-consumer --parallel --use-gaia --gaia-tag $(GAIA_TAG)

```

Reply:
Makefile should be adapted already to the  changes